--- a/ext/newblock/BlockMultiSided.java
+++ b/ext/newblock/BlockMultiSided.java
@@ -9,7 +9,7 @@
 
 public class BlockMultiSided extends ExtBlock {
 	private int field_1_2374;
-	private int sprite;
+	private int field_1_sprite;
 	private int field_1_2371;
 	private int field_1_2372;
 
@@ -18,7 +18,7 @@
 		this.sprite = sprite;
 		this.setTicksRandomly(true);
 		this.field_1_2374 = i2;
-		this.sprite = sprite;
+		this.field_1_sprite = sprite;
 		this.field_1_2371 = i4;
 		this.field_1_2372 = id;
 	}
@@ -28,13 +28,13 @@
 		this.sprite = sprite;
 		this.setTicksRandomly(true);
 		this.field_1_2374 = i2;
-		this.sprite = sprite;
+		this.field_1_sprite = sprite;
 		this.field_1_2371 = i4;
 		this.field_1_2372 = id;
 	}
 
 	public int getSprite(int face) {
-		return face == 1 ? this.field_1_2374 : (face == 0 ? this.field_1_2371 : this.sprite);
+		return face == 1 ? this.field_1_2374 : (face == 0 ? this.field_1_2371 : this.field_1_sprite);
 	}
 
 	public void tick(World world, int x, int y, int z, Random random) {
