--- net/minecraft/client/render/world/DirtyChunkSorter.java
+++ net/minecraft/client/render/world/DirtyChunkSorter.java
@@ -20,7 +20,8 @@
 		return this.method_1_1383(var1, var2);
 	}
 
-	public int compare(Object var1, Object var2) {
-		return this.423jUv2mCWNLqN9aM21K4tdzk250uS6p6u269SeaR118Ru5PNZ8sYufbxG0N((RenderChunk)var1, (RenderChunk)var2);
+	@Override
+	public int compare(RenderChunk var1, RenderChunk var2) {
+		return this._23jUv2mCWNLqN9aM21K4tdzk250uS6p6u269SeaR118Ru5PNZ8sYufbxG0N(var1, var2);
 	}
 }
--- ext/entity/SquidGames.java
+++ ext/entity/SquidGames.java
@@ -13,7 +13,7 @@
 import ext.util.ResourceLocation;
 import net.minecraft.entity.Entity;
 
-public class SquidGames<T extends 4GWFbOIMR3rg69M1Tw7AsIuO5A54yNk6ixz2D7h14bcq6qKB3v03aoGd8614DGBeJF7H1Bel756g1YCxyfkR0pQsk5P6wYjVNy> extends EntityModel<T> {
+public class SquidGames<T extends Entity> extends EntityModel<T> {
 	public static final ModelLayerLocation field_1_1713 = new ModelLayerLocation(new ResourceLocation("modid", "kill_squid"), "main");
 	private final ModelPart bb_main;
 
@@ -35,7 +35,8 @@
 		this.bb_main.render(var1, var2, var3, var4, var5, var6, var7, var8);
 	}
 
-	public void setupAnim(Object var1, float var2, float var3, float var4, float var5, float var6) {
-		this.2jXl38P4PmpdrRVSHS75EFrT141VO1F9V1Br9d1g60xH89Dyc0l9QeIIZP6q((Entity)var1, var2, var3, var4, var5, var6);
+	@Override
+	public void setupAnim(T var1, float var2, float var3, float var4, float var5, float var6) {
+		this._jXl38P4PmpdrRVSHS75EFrT141VO1F9V1Br9d1g60xH89Dyc0l9QeIIZP6q((T) var1, var2, var3, var4, var5, var6);
 	}
 }
--- ext/client/renderer/model/ModelCatBomb.java
+++ ext/client/renderer/model/ModelCatBomb.java
@@ -14,7 +14,7 @@
 import ext.util.ResourceLocation;
 import net.minecraft.entity.Entity;
 
-public class ModelCatBomb<T extends 4GWFbOIMR3rg69M1Tw7AsIuO5A54yNk6ixz2D7h14bcq6qKB3v03aoGd8614DGBeJF7H1Bel756g1YCxyfkR0pQsk5P6wYjVNy> extends EntityModel<T> {
+public class ModelCatBomb<T extends Entity> extends EntityModel<T> {
 	public static final ModelLayerLocation LAYER_LOCATION = new ModelLayerLocation(new ResourceLocation("modid", "catbomb"), "main");
 	private final ModelPart bb_main;
 
@@ -36,7 +36,8 @@
 		this.bb_main.render(var1, var2, var3, var4, var5, var6, var7, var8);
 	}
 
-	public void setupAnim(Object var1, float var2, float var3, float var4, float var5, float var6) {
-		this.07lo0Fq87IJA4mAYHw7xp5FVy3jZ5U6aoXY1czpa96akSwZO48eUwpsWeNE8((Entity)var1, var2, var3, var4, var5, var6);
+	@Override
+	public void setupAnim(T var1, float var2, float var3, float var4, float var5, float var6) {
+		this._7lo0Fq87IJA4mAYHw7xp5FVy3jZ5U6aoXY1czpa96akSwZO48eUwpsWeNE8((T) var1, var2, var3, var4, var5, var6);
 	}
 }
--- ext/world/zm/ZMWorld.java
+++ ext/world/zm/ZMWorld.java
@@ -116,7 +116,7 @@
 						}
 
 						public int compare(Object var1, Object var2) {
-							return this.65J8FrMXEvjjmSrGPd004RF9bPBQ25mQnZNsbDLRwV9Ny1Pr5XTIvmLF571p((WorldZMSpawn)var1, (WorldZMSpawn)var2);
+							return this._5J8FrMXEvjjmSrGPd004RF9bPBQ25mQnZNsbDLRwV9Ny1Pr5XTIvmLF571p((WorldZMSpawn)var1, (WorldZMSpawn)var2);
 						}
 					});
 					return var2.size() == 0 ? null : (var2.size() == 1 ? (WorldZMSpawn)var2.get(0) : (WorldZMSpawn)var2.get(this.random.nextInt(Math.min(var2.size(), 3))));
--- ext/world/gen/WorldGenHouseBigInterior.java
+++ ext/world/gen/WorldGenHouseBigInterior.java
@@ -34,7 +34,7 @@
 		this.field_1_2695 = var7;
 	}
 
-	public <T extends 8W1WSL13Q4ln33n7y62avj4L856TJi97cz688GPkUp1w7RteH2Xmt1Sz81L5765Ka5FhopL16lf9xj7Suw2rH92Al868XGr7> T method_1_1806(Class<T> var1) {
+	public <T extends IHouseBig> T method_1_1806(Class<T> var1) {
 		IHouseBig[] var2 = this.field_1_2698;
 		int var3 = var2.length;
 
--- ext/entity/EntityUnknownClass1.java
+++ ext/entity/EntityUnknownClass1.java
@@ -13,7 +13,7 @@
 import ext.util.ResourceLocation;
 import net.minecraft.entity.Entity;
 
-public class EntityUnknownClass1<T extends 4GWFbOIMR3rg69M1Tw7AsIuO5A54yNk6ixz2D7h14bcq6qKB3v03aoGd8614DGBeJF7H1Bel756g1YCxyfkR0pQsk5P6wYjVNy> extends EntityModel<T> {
+public class EntityUnknownClass1<T extends Entity> extends EntityModel<T> {
 	public static final ModelLayerLocation field_1_834 = new ModelLayerLocation(new ResourceLocation("modid", "custom_model"), "main");
 	private final ModelPart modelHead;
 	private final ModelPart modelBody;
@@ -55,7 +55,8 @@
 		this.modelLeftLeg.render(var1, var2, var3, var4, var5, var6, var7, var8);
 	}
 
-	public void setupAnim(Object var1, float var2, float var3, float var4, float var5, float var6) {
-		this.16PQb7B3oaJk6gI741J6hEWDKMl5ptux02FhX8e0iDi934fKn0uQ89vZiRom((Entity)var1, var2, var3, var4, var5, var6);
+	@Override
+	public void setupAnim(T var1, float var2, float var3, float var4, float var5, float var6) {
+		this._6PQb7B3oaJk6gI741J6hEWDKMl5ptux02FhX8e0iDi934fKn0uQ89vZiRom((T) var1, var2, var3, var4, var5, var6);
 	}
 }
--- net/minecraft/client/render/world/DistanceChunkSorter.java
+++ net/minecraft/client/render/world/DistanceChunkSorter.java
@@ -18,7 +18,8 @@
 		return this.method_1_1270(var1, var2);
 	}
 
-	public int compare(Object var1, Object var2) {
-		return this.09t8zMS9yrC79Db6494p93jjD4H1QX7P3t710r95O3cClfkpQuZo8To95RiL((RenderChunk)var1, (RenderChunk)var2);
+	@Override
+	public int compare(RenderChunk var1, RenderChunk var2) {
+		return this._9t8zMS9yrC79Db6494p93jjD4H1QX7P3t710r95O3cClfkpQuZo8To95RiL(var1, var2);
 	}
 }
--- net/minecraft/crafting/CraftingManager__57536588.java
+++ net/minecraft/crafting/CraftingManager__57536588.java
@@ -19,7 +19,8 @@
 		return this.method_1_2075(var1, var2);
 	}
 
-	public int compare(Object var1, Object var2) {
-		return this.8G5dJ7pdAEvNRP6jYct7bw8mLKn02Ks5P34Q7tGoJaoWCx1lY3Jju44OjKl6((Recipe)var1, (Recipe)var2);
+	@Override
+	public int compare(Recipe var1, Recipe var2) {
+		return this._G5dJ7pdAEvNRP6jYct7bw8mLKn02Ks5P34Q7tGoJaoWCx1lY3Jju44OjKl6(var1, var2);
 	}
 }
--- net/minecraft/client/resource/ResourceDownloader.java
+++ net/minecraft/client/resource/ResourceDownloader.java
@@ -65,15 +65,15 @@
 			}
 
 			public FileVisitResult postVisitDirectory(Object var1, IOException var2) throws IOException {
-				return this.6Y066QkhR7VJ68J5178YcHIRIq6YMqDPYIin6Vv049v71311GMTvI0DN2U92((Path)var1, var2);
+				return this._Y066QkhR7VJ68J5178YcHIRIq6YMqDPYIin6Vv049v71311GMTvI0DN2U92((Path)var1, var2);
 			}
 
 			public FileVisitResult visitFileFailed(Object var1, IOException var2) throws IOException {
-				return this.23MZBIGCTSC3NUicInCEvwRdy4FAJix3YthzrA5QlBOEkmlz7Sg7u66bTfh6((Path)var1, var2);
+				return this._3MZBIGCTSC3NUicInCEvwRdy4FAJix3YthzrA5QlBOEkmlz7Sg7u66bTfh6((Path)var1, var2);
 			}
 
 			public FileVisitResult visitFile(Object var1, BasicFileAttributes var2) throws IOException {
-				return this.2rJuaeK1zQ1N7IGF5Go9hj61bHOKKocGJwAenj466SVemdP80Nt2ZlEB7B1v((Path)var1, var2);
+				return this._rJuaeK1zQ1N7IGF5Go9hj61bHOKKocGJwAenj466SVemdP80Nt2ZlEB7B1v((Path)var1, var2);
 			}
 		};
 		Files.walkFileTree(var1, var2);
--- ext/entity/EntityUnknownClass2.java
+++ ext/entity/EntityUnknownClass2.java
@@ -13,7 +13,7 @@
 import ext.util.ResourceLocation;
 import net.minecraft.entity.Entity;
 
-public class EntityUnknownClass2<T extends 4GWFbOIMR3rg69M1Tw7AsIuO5A54yNk6ixz2D7h14bcq6qKB3v03aoGd8614DGBeJF7H1Bel756g1YCxyfkR0pQsk5P6wYjVNy> extends EntityModel<T> {
+public class EntityUnknownClass2<T extends Entity> extends EntityModel<T> {
 	public static final ModelLayerLocation field_1_1428 = new ModelLayerLocation(new ResourceLocation("modid", "observer"), "main");
 	private final ModelPart model;
 
@@ -35,7 +35,8 @@
 		this.model.render(var1, var2, var3, var4, var5, var6, var7, var8);
 	}
 
-	public void setupAnim(Object var1, float var2, float var3, float var4, float var5, float var6) {
-		this.5u9iRyfW4CjyP115I6aD5UN5KOFCIx98GeMk50Lbn16CK14CUJj8Og4n71Sl((Entity)var1, var2, var3, var4, var5, var6);
+	@Override
+	public void setupAnim(T var1, float var2, float var3, float var4, float var5, float var6) {
+		this._u9iRyfW4CjyP115I6aD5UN5KOFCIx98GeMk50Lbn16CK14CUJj8Og4n71Sl((T) var1, var2, var3, var4, var5, var6);
 	}
 }
